### PR TITLE
Adding sass and themes to the project

### LINKED ISSUE #19: As a developer I would like to use SCSS in the codebase

### PR DESCRIPTION
**Pull request by esteeroramy**
Fixes #19

### CODE REVIEW

File: sass/css/app.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import 'themes/themes.scss';
+@import 'themes/themify.scss';
+@import 'themes/themed.scss';
```
Comment (SergeyGV) [inline sass/css/app.scss]: Is it possible to diversify these file names a bit? Having two files of 'themes' and 'themed' is somewhat confusing

File: velocity.js [hunk 2]
```diff
@@ -26,7 +29,16 @@ app.use('/jquery', express.static(`${__dirname}/node_modules/jquery/dist/`));
...
+app.use(
+    sassMiddleware({
+        src: `${__dirname}/sass`, 
+        dest: `${__dirname}/UI/stylesheets`,
+        prefix:  '/stylesheets',
+        debug: true,
+        outputStyle: 'compressed'
+    })
+);
+app.use(express.static(path.join(__dirname, 'UI')));
```
Comment (ahmedqarmout) [inline velocity.js]: path.join(__dirname, 'UI') -> `${__dirname}/UI`

File: velocity.js [hunk 3]
```diff
@@ -3,7 +3,10 @@ const bodyParser = require('body-parser');
 const config = require(`${__dirname}/Backend/config.js`);
 const express = require('express');
 const i18n = require("i18n");
+const path = require('path');
```
Comment (ahmedqarmout) [inline velocity.js]: take this one out plz

File: velocity.js [hunk 4]
```diff
@@ -26,7 +29,16 @@ app.use('/jquery', express.static(`${__dirname}/node_modules/jquery/dist/`));
 app.use('/bootstrap', express.static(`${__dirname}/node_modules/bootstrap/dist`));
 app.use('/materializecss', express.static(`${__dirname}/node_modules/materialize-css/dist`));
 app.use('/animate', express.static(`${__dirname}/node_modules/animate.css/`));
-app.use(express.static(`${__dirname}/UI`));
+app.use(
+    sassMiddleware({
+        src: `${__dirname}/sass`, 
+        dest: `${__dirname}/UI/stylesheets`,
+        prefix:  '/stylesheets',
+        debug: true,
```
Comment (ahmedqarmout) [inline velocity.js]: what does this do?

File: views/layout.pug [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
-      title Velocity
+        block styles
+        block title
+            title Velocity
 
-  body
-    block content
-    block scripts
\ No newline at end of file
+    body.theme-default
```
Comment (ahmedqarmout) [inline views/layout.pug]: we need this as user.theme, but we can leave this for another PR

File: views/login.pug [hunk 6]
```diff
@@ -1,13 +1,12 @@
 extends layout
 
 block styles
-    link(rel='stylesheet' href='/css/login.css')
+    link(rel='stylesheet' href='/stylesheets/css/login.css')
 
 block title
     title Velocity - Login
```
Comment (ahmedqarmout) [inline views/login.pug]: move this to translation file plz

File: views/modeSelector.pug [hunk 7]
```diff
@@ -1,7 +1,7 @@
 extends layout
 
 block styles
-    link(rel='stylesheet' href='/css/modeSelector.css')
+    link(rel='stylesheet' href='/stylesheets/modeSelector.css')
 
 block title
     title Select mode
```
Comment (ahmedqarmout) [inline views/modeSelector.pug]: translation file plz

File: sass/css/app.scss (hunk 1 above)
Comment (ahmedqarmout) [reply sass/css/app.scss]: which one of them we are actually using? if we will need to use all 3 then we need better names, but if we are only using one as a gateway for the other, then its fine

File: views/login.pug (hunk 6 above)
Comment (esteeroramy) [reply views/login.pug]: good catch

File: views/modeSelector.pug (hunk 7 above)
Comment (esteeroramy) [reply views/modeSelector.pug]: good catch

File: views/layout.pug (hunk 5 above)
Comment (esteeroramy) [reply views/layout.pug]: i got uu
